buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java

```diff
@@ -153,16 +155,29 @@ private static void addScmInfo(XmlProvider xml) {
         for (final Object child : root.children()) {
             if (child instanceof Node) {
                 final Node node = (Node) child;
-                if (node.name() instanceof QName) {
-                    final QName qname = (QName) node.name();
-                    if (qname.matches("url")) {
-                        url = node;
-                    } else if (qname.matches("scm")) {
-                        scm = node;
+                final Object name = node.name();
+
+                try {
+                    // For Gradle 6.8 and below, the class is groovy.xml.QName
+                    // For Gradle 7.4 and above, the class is groovy.namespace.QName
+                    if (name != null && name.getClass().getSimpleName().equals("QName")) {
```

Comment:
Do we still have to support Gradle 6.x ?  Do you still see value? 
I believe all our components moved to Gradle 7+.

Reply:
Yes, that's basically on plugin authors ... We support it